### ISSUE TITLE
Idle kick avoidance and W10

### ISSUE THREAD

**Issue by killj0yyy** [Pure-Battlefield/PureSeeder3#10]
When Idle kick avoidance runs in Windows 10 it causes the game to be primary window and go full screen. It is no longer a silent process. 

Windows 8.1 works just fine. I am on the latest November update for W10.

**Comment by SKHecht** [comment]
I'm currently in the middle of finals. After next week I'll look into this.

**Comment by ForceTen2112** [comment]
So you know, I have not forgotten about this and am currently looking into the issue.

**Comment by killj0yyy** [comment]
Thanks the both of you. Are you able to reproduce the issue?

**Comment by ForceTen2112** [comment]
Haha, actually, these are just both of my Github accounts and I frequently forget which one I am signed in on. And I am not sure that I am reproducing it (although, admittedly, I am not that familiar with how the seeder is intended to run). What I am seeing is that while I use another application, like Chrome, I can use it uninterrupted besides for the occasionally flickering of BF; but when I minimize the application and there are no other applications to take control of the screen, BF takes over. I do notice that BF doesn't seem to be minimized be default, which I will look into.

**Comment by killj0yyy** [comment]
I have it running right now.. I assume it passes key presses every 10 seconds? For the most part every 10 seconds or so it'll kick me back into the game. Sometimes It will work okay, others not at all.

**Comment by ForceTen2112** [comment]
Yeah, it clicks in the window and then waits 2 minutes (based on the code, but it could be less). So ever 10 or so seconds Battlefield pops up and takes over your screen? What else are you doing at the time?

**Comment by killj0yyy** [comment]
Just browsing the Internet using chrome. How often does it click in the
window?

**Comment by AndriyYer** [comment]
Banned from PURE over two years ago, still get these emails.

Hah

**Comment by ForceTen2112** [comment]
This is actually just the PURE GitHub repo, not technically connected to PURE the organization. If you don't want to receive notifications from this repo you need to unsubscribe.
